src/pages/home/report/PureReportActionItem.tsx

```diff
@@ -296,7 +296,7 @@ function PureReportActionItem({
 }: PureReportActionItemProps) {
     const {translate} = useLocalize();
     const {shouldUseNarrowLayout} = useResponsiveLayout();
-    const reportID = report?.reportID ?? '';
+    const reportID = report?.reportID ?? `${CONST.DEFAULT_NUMBER_ID}`;
```

Comment:
@daledah Why this changes are in this PR? Do main does not have this changes already, I assume when eslint rule is added all the refactoring is also made to app

Reply:
https://expensify.slack.com/archives/C01GTK53T8Q/p1734427537784129 so this is a new eslint which effects a lot of places. It looks like everyone is handling it in their own PR. @daledah Let's update the code according to eslint rule only in the file we are meant to originally change.